src/ui/__tests__/ui_i18n.js

```diff
@@ -0,0 +1,123 @@
+import expect from 'expect.js';
```

Comment:
gonna hold off on reviewing this file since I asked for some major changes to the module it's testing.